.github/workflows/test_wheels.yml

```diff
@@ -1,12 +1,12 @@
 name: test wheel builds
 
 on:
-  pull_request_review_comment:
-    types: [created, edited]
+  pull_request:
+    types: [labeled]
```

Comment:
Did not know this is possible nice find.

Reply:
:D